src/content/articles/alab.md

```diff
@@ -88,30 +88,47 @@ downloadSection: {
   # The download links and button setup for the download table.
   downloads: [{
     buttonText: "DOWNLOAD",
-    downloadUrl: "https://dpel-assets.aswf.io/usd-alab/alab-v2.1.0.zip",
+    downloadUrl: "https://dpel-assets.aswf.io/usd-alab/alab-v2.2.0.zip",
```

Comment:
This URL will change, once we create a release of DPEL/ALab, it will point to the generated zip on GitHub.

Reply:
The final URL from the GitHub release:
https://github.com/DigitalProductionExampleLibrary/ALab/archive/refs/tags/v2.2.0.zip

And its final size is 294 MB